### PR TITLE
Lite obfuscation

### PR DESCRIPTION
**Pull request by babenek**
- applied standard logging with debug level
- use lite obfuscation to skip PEM key obfuscation for strictly check
- prepare to use FileId in file name instead unpredictable integer of order in custom list

### CODE REVIEW

File: snapshot.yaml [hunk 1]
```diff
@@ -1175,3 +1175,9 @@
   tag: v6.1.5-9-gf6fb94a9
   url: https://github.com/GoogleChrome/workbox
 
+
+
+
+
+
+
```
Comment (csh519) [inline snapshot.yaml:1183]: Ooops? :eyes:

**Review by csh519 — APPROVED** [review]
Lite obfuscation of directory is applied.

LGTM :+1:

File: snapshot.yaml (hunk 1 above)
Comment (babenek) [reply snapshot.yaml:1183]: The change is required to trigger the job in github.

**Review by iuriimet — APPROVED** [review]

**Comment by kmnls** [comment]
> Lite obfuscation of directory is applied.
> 
> LGTM 👍

Is it acceptable to let skip obfuscation of keys?
Lets assume that checked repo is closed and accessible only to CredSweeper. When you download its key into the system you may expose it to other users of CredSweeper even if they do not have direct access to the vulnerable repo. It seems to me that it may lead to data breach.

**Comment by babenek** [comment]
> > Lite obfuscation of directory is applied.
> > LGTM 👍
> 
> Is it acceptable to let skip obfuscation of keys? Lets assume that checked repo is closed and accessible only to CredSweeper. When you download its key into the system you may expose it to other users of CredSweeper even if they do not have direct access to the vulnerable repo. It seems to me that it may lead to data breach.

Only open data is used for the benchmark

**Comment by csh519** [comment]
> > > Lite obfuscation of directory is applied.
> > > LGTM 👍
> > 
> > 
> > Is it acceptable to let skip obfuscation of keys? Lets assume that checked repo is closed and accessible only to CredSweeper. When you download its key into the system you may expose it to other users of CredSweeper even if they do not have direct access to the vulnerable repo. It seems to me that it may lead to data breach.
> 
> Only open data is used for the benchmark

I discussed about this with others and I just realized it can be a problem regarding leaking credentials.
Following the previous mail of privacy data office it can be a problem :(
So I just opened revert PR(#54) of this and please review it quickly and merge.

**Comment by csh519** [comment]
How about add custom PEM Key created by our side?
As I see there is a few cases of FN case in PEM Key rule.

**Pull request merged**